test/apiTest.js

```diff
@@ -117,7 +132,7 @@ describe('POST /cats', function () {
             .post('/cats')
             .send(body)
             .set('Accept', 'application/x-www-form-urlencoded')
-            // .expect('Content-Type', /json/) // API Test Error: got "text/html; charset=utf-8"
+            // .expect('Content-Type', /json/) // API Test Error: got "text/html; charset=utf-8" // FIXIT - what is going on here?
```

Comment:
the convention is to use fixme - quite often you'll get special syntax highlighting in editors and the like :)